dockerfiles/run/guice/cassandra/destination/conf2/dnsservice.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one   
+  or more contributor license agreements.  See the NOTICE file 
+  distributed with this work for additional information        
+  regarding copyright ownership.  The ASF licenses this file   
+  to you under the Apache License, Version 2.0 (the            
+  "License"); you may not use this file except in compliance   
+  with the License.  You may obtain a copy of the License at   
+                                                               
+    http://www.apache.org/licenses/LICENSE-2.0                 
+                                                               
+  Unless required by applicable law or agreed to in writing,   
+  software distributed under the License is distributed on an  
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       
+  KIND, either express or implied.  See the License for the    
+  specific language governing permissions and limitations      
+  under the License.                                           
+ -->
+
+<!-- Read https://james.apache.org/server/config-dnsservice.html for further details -->
+
+<dnsservice>
+  <servers>
+    <server>8.8.8.8</server>
```

Comment:
Do we want to use google dns ? :trollface:

Reply:
Maybe not..? I noticed the other day though that all our `dnsservice.xml` have that dns declared :trollface:

Reply:
I didn't know it was there :) I had a hard time makes two dockers james can recognize each other's domain through DNS